decred_wallet/view_controller/SendViewController.swift

```diff
@@ -1,11 +1,9 @@
 //
 //  SendViewController.swift
 //  Decred Wallet
-
 // Copyright (c) 2018-2019 The Decred developers
 // Use of this source code is governed by an ISC
 // license that can be found in the LICENSE file.
-
```

Comment:
An empty line space here helps to separate the multi-line comment above from the beginning of code instructions, it's needed. Please restore it.